core/src/storage2/collections/hashmap/tests.rs

```diff
@@ -22,6 +22,63 @@ use crate::{
 };
 use ink_primitives::Key;
 
+#[cfg(feature = "ink-fuzz")]
```

Comment:
I think it is better if we move all `ink-fuzz` dependent fuzz tests into their own file so that we can have only one `#[cfg(feature = "ink-fuzz)]` and do not spill those conditional compilation flags around the entire code base. Also I'd like to rename `ink-fuzz` into `ink-fuzz-tests`.